improver/utilities/spatial.py

```diff
@@ -761,3 +761,55 @@ def add_vicinity_coordinate(
         point, units=units, long_name="radius_of_vicinity", attributes=attributes
     )
     cube.add_aux_coord(coord)
+
+
+def maximum_in_height(
```

Comment:
I'm not sure this is the best place for this new method. All the other methods in `spatial.py` deal with horizontal manipulations. The other possibility is in `cube_manipulation.py`, which contains other collapsing methods - I don't think you want to use `collapsed()` though as it is designed to override Iris' metadata, which are right for this method.

Reply:
I've taken your suggestions and moved the plugin and tests etc. to cube_manipulation.py